### PR TITLE
[P2] Enabled support for abandoned carts.

### PR DESCRIPTION
**Pull request by brayanblanco**
This change let us use Api2cart queries to find carts based on modified and created date to enhance the connection with Engagement Cloud.

##How to test.
 - Replace the files in your plugin folder
 - Deactivate and reactivate the plugin (so the api2cart flags get loaded up)
 - Add/remove items to a cart.
 - Check in Api2Cart store that the abandoned list of orders have populated the created and modified dates and that they can be used in the query (`modified/created_from/to` format).

### CODE REVIEW

**Comment by brayanblanco** [comment]
BEfore to merge we have to wait to A2C to see what's their requirements on the created date (i,e, they don't use the same format as in modified... perhaps the a2c version changes as well...).

File: includes/class-engagement-cloud-loader.php [hunk 1]
```diff
@@ -122,5 +122,21 @@ public function run() {
...
+		
+		$this->enable_a2c();
+	}
+	
+	/**
+	 * Set the options so Api2Cart cam know that modified and created date queries are supported
+	 *
+	 * @since    1.1.0
+	 */
+	private function enable_a2c(){
```
Comment (DannyRendle) [inline includes/class-engagement-cloud-loader.php]: This is the only abbreviation of API2Cart. Can we standardise it so all references are or aren't? I vote for aren't.

Comment (brayanblanco) [reply includes/class-engagement-cloud-loader.php]: I agree

**Comment by andyp159** [comment]
This works well. Should we up the version number of the plugin though? It was appearing on my machine as the same version numer as live :|

**Comment by andyp159** [comment]
@brayanblanco - I've just noticed that, when downloading, the zip file has the original name - before the rebrand. Can this be amended so it matches the current version?

**Pull request merged**